kola/tests/misc/network.go

```diff
@@ -251,3 +306,11 @@ func wireguard(c cluster.TestCluster) {
 	m := c.Machines()[0]
 	c.AssertCmdOutputContains(m, `ip --json address show kv0 | jq -r '.[] | .addr_info | .[] | select( .family == "inet") | .local'`, "10.100.0.2")
 }
+
+// iptables save and restore iptables rules.
+func firewall(c cluster.TestCluster) {
+	m := c.Machines()[0]
+	// We test that the ruleset has been created using iptables or nftables.
+	// This would return an error if the ruleset is not created.
+	c.AssertCmdOutputContains(m, `sudo nft --json list ruleset | jq '.nftables | .[] | select(.chain) | .chain.policy'`, "accept")
```

Comment:
Isn't the default to accept? Shouldn't we check for the port 80 drop rule?

Reply:
Okay. It can be simplified slightly.

```suggestion
	c.AssertCmdOutputContains(m, `sudo nft --json list ruleset | jq '.nftables.[] | select(.chain) | .chain.policy'`, "accept")
```